webapp/app/forms/steps/step.py

```diff
@@ -66,15 +66,16 @@ def __init__(self, title, form, intro=None, prev_step=None, next_step=None, head
         self.form = form
         self.template = template
 
-    def create_form(self, request, prefilled_data):
+    def create_form(self, form_data=None, prefilled_data=None):
         # If `form_data` is present it will always override `data` during
         # value binding. For `BooleanFields` an empty/missing value in the `form_data`
         # will lead to an unchecked box.
-        form_data = request.form
-        if len(form_data) == 0:
-            form_data = None
 
-        form = self.form(form_data, **prefilled_data)
+        extracted_form_data = request.form  # Override the form_data for old steps
+        if len(extracted_form_data) == 0:
+            extracted_form_data = None
+
+        form = self.form(extracted_form_data, **prefilled_data)
```

Comment:
My explanation: These are the old steps and we do not set form_data in the multistep_flow. However, because they will be replaced by the SteuerlotseSteps, we do not want to change that too much.